package.json

```diff
@@ -11,7 +11,9 @@
     "@types/node": "^16.18.11",
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
+    "bootstrap": "^5.3.0",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.7.4",
```

Comment:
Can you double-check that `react-bootstrap` requires the additional `bootstrap` dependency? Does it rely on it or does the `react-bootstrap` dependency have everything baked in already?

Reply:
We talked about just using the `bootstrap CDN` for now and if we need to add some flare later down the road, we can always install `bootstrap` iif we need access to the css files.